docs/configuring.md

```diff
@@ -721,6 +721,15 @@ Suffix that instructs which files are to be processed by Jinja as templates.
     _templates_suffix: .my-custom-suffix
     ```
 
+Starting at version 6, an empty suffix is also valid, and will instruct Copier to copy
```

Comment:
```suggestion
An empty suffix is also valid, and will instruct Copier to copy
```


Version changes details belong to the Changelog (that's what I meant by "roadmap", Sorry for the mistake 😅).